cancer_ai/validator/competition_handlers/base_handler.py

```diff
@@ -15,15 +27,12 @@ class ModelEvaluationResult(BaseModel):
     fpr: list = []
     tpr: list = []
     roc_auc: float = 0.0
-    run_time_s: float = 0.0
-    tested_entries: int = 0
-
-    score: float = 0.0
 
-    error: str = ""
 
-    class Config:
-        arbitrary_types_allowed = True
+class TricorderEvaluationResult(BaseModelEvaluationResult):
+    accuracy: float = 0.0
```

Comment:
i know we dont know yet but shouldnt accuracy be part of base model most likely?
if we are not sure i would add some todo comment to adjust it when we will know etc.

Reply:
accuracy is always :)

Reply:
yeah so lets move it to base model then